### PR TITLE
[ipa] update logs to be collected

### PR DESCRIPTION
**Pull request by MartinBasti**
Several logs have been missing in ipa plugin. This PR adds them.

### CODE REVIEW

File: sos/plugins/ipa.py [hunk 1]
```diff
@@ -108,6 +108,7 @@ def setup(self):
             "/var/log/krb5kdc.log",
             "/var/log/dirsrv/slapd-*/logs/access",
             "/var/log/dirsrv/slapd-*/logs/errors",
+            "/var/log/httpd/error_log",
```
Comment (bmr-cymru) [inline sos/plugins/ipa.py]: This is already collected in the `apache` plugin - if IPA is using it for API access then the best option is probably to add `apache` to the existing `identity` profile.

**Comment by bmr-cymru** [comment]
Hi Martin, thanks for the pull request. Please could you add a 'Signed-off-by' (DCO) to your patches, and put the commit subject in the `[plugin] short description of change` format required by the [Contributor Guidelines](https://github.com/sosreport/sos/wiki/Contribution-Guidelines) and we can get this reviewed and merged?

No need to open a new pull request - just amend the existing commits (or edit them with `git rebase -i`), and then re-push to the same branch with `git push --force`.

Thanks,

**Comment by MartinBasti** [comment]
Thank you, I updated commits as you requested

File: sos/plugins/ipa.py [hunk 2]
```diff
@@ -87,7 +91,11 @@ def setup(self):
 
             self.add_copy_spec([
                 "/var/log/ipaserver-install.log",
-                "/var/log/ipareplica-install.log"
+                "/var/log/ipaserver-kra-install.log",
+                "/var/log/ipareplica-install.log",
+                "/var/log/ipareplica-install.log",
```
Comment (tomaskrizek) [inline sos/plugins/ipa.py]: This log is already included by the previous line.

Comment (MartinBasti) [inline sos/plugins/ipa.py]: fixed thanks

**Comment by MartinBasti** [comment]
Bump for review

**Comment by tscherf** [comment]
Is there anything needed before this PR can be merged?

**Comment by pmoravec** [comment]
Code changes seem OK, just a minor comment to commit message(s) - they should refer to the issue / PR, like having there a line like:

    Resolves: #1010 

or:

    Closes: #1010 

or:

    Related to: #1010

**Comment by MartinBasti** [comment]
Commit messages amended

**Comment by tscherf** [comment]
Bump for review

**Comment by pmoravec** [comment]
+1 to the PR

**Comment by tscherf** [comment]
bump for review

**Pull request closed without merging**